run.py

```diff
@@ -643,7 +648,7 @@ def fetch_test_details(var) -> dict:
     total_run_time = (datetime.datetime.now() - run_start_time)
     total_time = """'''Start Time:''' {a} '''End Time:''' {b} '''Total duration:'''{c} \
```

Comment:
while this is not related to this PR. correcting this would be better for readability. spit into different rows and add sub rows or columns in this one itself.

Reply:
but you can choose to dismiss this if you want to take this in a separate PR

Reply:
Added a separate section with the timing details